### PR TITLE
ci: Remove local dart dependency in flutter CI jobs

### PR DESCRIPTION
**Pull request by mtrezza**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

### Issue Description

Closes: https://github.com/parse-community/Parse-SDK-Flutter/issues/832

### Approach

Remove automatic execution of developer script that uses the local dart package instead of the released dart package.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,12 +46,10 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           cache: true
-      - name: Set file permissions
-        run: chmod +x tools/flutter-dependencies.sh
       - name: Install dependencies
         run: |
-          echo "publish_to: none" >> packages/flutter/pubspec.yaml
-          tools/flutter-dependencies.sh
```
Comment (mbfakourii) [inline .github/workflows/ci.yml:54]: Should the flutter-dependencies.sh file be deleted from the tools folder?

**Review by mbfakourii — APPROVED** [review]
I checked the file, I think there is no problem. Must be tested

**Comment by mtrezza** [comment]
Ok, I'll go ahead and merge and we'll see.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mtrezza) [reply .github/workflows/ci.yml:54]: I’ve kept it in case it serves any purpose when developing locally.

Comment (mtrezza) [reply .github/workflows/ci.yml:54]: Should I remove the 2 script files?

Comment (mbfakourii) [reply .github/workflows/ci.yml:54]: No, it is not forced to delete.
But the developer can also run the code inside the script manually

Comment (mtrezza) [reply .github/workflows/ci.yml:54]: Alright so let's keep it for now

**Pull request merged**